pages/api/forgetPassword.ts

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+import type { NextApiRequest, NextApiResponse } from 'next';
+
+export default async (req: NextApiRequest, res: NextApiResponse) => {
+    try {
+        await axios({
+            method: 'post',
+            url: 'https://mysejahtera.malaysia.gov.my/register/forgotPassword',
+            data: {
+                emailOrUserName: req.body.emailOrUserName
```

Comment:
`req.body.username` would be better, mysejahtera practise isn't the best...

Reply:
name changed in the commit of https://github.com/nubpro/nakvaksin/pull/5/commits/f25bfc959bbfd4f6af26d87d9e89e9722f1611be